psalm.xml

```diff
@@ -12,7 +12,9 @@
         <directory name="src" />
         <ignoreFiles>
             <directory name="src/Filesystem" />
+            <directory name="src/Auth" />
             <directory name="vendor" />
+            <file name="src/Controller/Component/AuthComponent.php" />
```

Comment:
Is it a good idea to ignore a whole file? Don't we might miss some other valuable ones?

Reply:
Deprecated classes aren't going to get any updates. All existing major issues are already fixed, so ignoring remaining noisy issues isn't a problem IMO.